src/libraries/System.IO.Compression/src/System/IO/Compression/ZipHelper.cs

```diff
@@ -189,5 +190,15 @@ private static bool SeekBackwardsAndRead(Stream stream, byte[] buffer, out int b
                 return true;
             }
         }
+
+        internal static bool GetSuppressValidation()
+        {
+            if (AppContext.TryGetSwitch("System.Compression.ZipArchiveEntry.SuppressValidation", out bool suppressValidation))
```

Comment:
Should this be named `SuppressHeaderValidation`? Or is this the only kind of validation we would ever want to suppress? Or would we only ever want 1 "suppress validation" switch?

Reply:
yeah `SuppressHeaderValidation` makes more sense